packages/web3/src/entitlement.test.ts

```diff
@@ -271,7 +271,8 @@ test.each(nftCases)('erc721Check - $desc', async (props) => {
     const controller = new AbortController()
     const result = await evaluateTree(controller, wallets, [provider], check)
     if (expectedResult) {
-        expect(result).toBeTruthy()
+        expect(result as Address).toBeTruthy()
```

Comment:
Change all test cases that expect an entitled wallet to check that the string result is non-zero address